src/boringssl/gen_build_yaml.py

```diff
@@ -164,5 +164,10 @@ def WriteFiles(self, files, asm_outputs):
 
   print yaml.dump(g.yaml)
 
+except OSError as err:
+  type, value, traceback = sys.exc_info()
+  raise RuntimeError, (
```

Comment:
The only problem with that is that when the exception crashes the program, it only prints `OSError: [Errno 2] No such file or directory`

The other solution I was thinking of was to add the more verbose error message by calling sys.stderr.write with a custom message, then letting the original exception continue.